ingest/lambdas/bulk-ingester.js

```diff
@@ -0,0 +1,18 @@
+// @ts-check
+const bulkIngester = require('../lib/bulk-ingester');
+
+const handler = async () => {
+  const dspaceEndpoint = process.env.DSPACE_ENDPOINT;
+  const ingestTopicArn = process.env.INGEST_TOPIC_ARN;
+  if (dspaceEndpoint === undefined || ingestTopicArn === undefined) {
```

Comment:
Let's make sure we also catch the case where it's set to an empty string.

```suggestion
  if (!dspaceEndpoint || !ingestTopicArn) {
```